README.md

```diff
@@ -1,2 +1,113 @@
+<a id="contents"></a>
+
 # i18n-static
 Translation editable A3 pieces with i18next
+
+1. [Installation](#1)<br>
+2. [Configuration](#2)<br>
+3. [Behavior](#3)<br>
+4. [Import and export](#4)<br>
+
+This module adds editable pieces for translation through i18next to an A3 project.
+
+> This module is intended to localize static text in templates i.e text wrapped with `__("...")`, **not localize editable content.** If your goal is content localization, you should use the [default localization feature in A3](https://v3.docs.apostrophecms.org/guide/localization/) instead.
+
+---
+
+JSON files used in Apostrophe are used by i18next. These resources are automatically converted as pieces through this lodule `@apostrophecms/i18n-static`.
+
+This will add an entry in the admin bar "I18n Static Phrases".
+
+Only admin users have access to this module for now.
+
+<a id="1"></a>
+
+## 1 Installation [&#x2B06;](#contents)
+
+Install the module with `npm i @apostrophecms/i18n-static` then declare it in the modules configuration in `app.js`:
+
+```js
+  modules: {
+    '@apostrophecms/i18n-static': {},
+  }
+```
+
+<a id="2"></a>
+
+
+## 2 Configuration [&#x2B06;](#contents)
+
+The locales configuration comes from the i18n module in a A3 project. Usually, in `modules/@apostrophecms/i18n/index.js`. Example:
+
+```js
+module.exports = {
+  options: {
+    defaultLocale: 'en',
+    locales: {
+      en: { label: 'English' },
+      fr: {
+        label: 'French',
+        prefix: '/fr'
+      },
+      sk: {
+        label: 'Slovak',
+        prefix: '/sk'
+      },
+      es: {
+        label: 'Spanish',
+        prefix: '/es'
+      },
+      'pt-BR': {
+        label: 'Portuguese',
+        prefix: '/pt'
+      },
+      ar: {
+        label: 'Arabic',
+        prefix: '/ar'
+      }
+    }
+  }
+};
+```
+
+I18n-static pieces have a `namespace` field. Namespaces are found in the A3 project. But if resources from specific namespaces should not be converted to i18n-static pieces, they can be excluded by configuring the array `excludeNamespaces` in the module's options.
+
+```js
+  modules: {
+    '@apostrophecms/i18n-static': {
+      options: {
+        excludeNamespaces: [ 'aposEvent', 'aposForm' ]
+      }
+    },
+  }
+```
+
+As the `autopublish` option is set to `true` by default, new pieces will be created as draft and live. Change this option to `false` if `draft` is necessary before publishing them later (the `publish` batch feature will be helpful in this case).
```

Comment:
This is a good place to first mention that the feature is edited via a piece type.

Reply:
No that works.